### ISSUE TITLE
Enums critical to xrt::bo sync behavior exist in deprecated header file

### ISSUE THREAD

**Issue by wilderfield** [Xilinx/XRT#7539]
https://github.com/Xilinx/XRT/blob/ca535535e3052f4157d84512b0be47f31aad8abd/src/runtime_src/core/include/deprecated/xrt.h#L176-L181

Should we make new enums, since these are depracated?

`XRT_BO_SYNC_BO_TO_DEVICE` for example?

Or maybe `xrt::bo::direction::to_device`

**Comment by stsoe** [comment]
Yes, this has to be fixed, the old style enums should not have been used to begin with.
